openpype/hosts/houdini/plugins/publish/validate_scene_review.py

```diff
@@ -37,36 +43,37 @@ def process(self, instance):
                 "\n\n".join(report),
                 title=self.label)
 
-    def get_invalid_scene_path(self, rop_node):
-        scene_path_parm = rop_node.parm("scenepath")
+    def get_invalid_scene_path(self, instance):
+
+        node = hou.node(instance.data.get("instance_node"))
+        scene_path_parm = node.parm("scenepath")
         scene_path_node = scene_path_parm.evalAsNode()
         if not scene_path_node:
-            path = scene_path_parm.evalAsString()
-            return "Scene path does not exist: '{}'".format(path)
+            return [scene_path_parm.evalAsString()]
 
-    def get_invalid_camera_path(self, rop_node):
-        camera_path_parm = rop_node.parm("camera")
-        camera_node = camera_path_parm.evalAsNode()
-        path = camera_path_parm.evalAsString()
-        if not camera_node:
-            return "Camera path does not exist: '{}'".format(path)
-        type_name = camera_node.type().name()
-        if type_name != "cam":
-            return "Camera path is not a camera: '{}' (type: {})".format(
-                path, type_name
-            )
+    def get_invalid_lop_network(self, instance):
+        node = hou.node(instance.data.get("instance_node"))
+        lop_network_parm = node.parm("loppath")
+        lop_network_node = lop_network_parm.evalAsNode()
+        node_name = lop_network_parm.evalAsString()
+        if not lop_network_node:
+            return [node_name]
+        lop_node = hou.node(node_name)
+        if lop_node.type().name() != "lopnet":
+            return [node_name]
 
-    def get_invalid_resolution(self, rop_node):
+    def get_invalid_resolution(self, instance):
+        node = hou.node(instance.data.get("instance_node"))
 
         # The resolution setting is only used when Override Camera Resolution
         # is enabled. So we skip validation if it is disabled.
-        override = rop_node.parm("tres").eval()
+        override = node.parm("tres").eval()
         if not override:
             return
 
         invalid = []
-        res_width = rop_node.parm("res1").eval()
-        res_height = rop_node.parm("res2").eval()
+        res_width = node.parm("res1").eval()
+        res_height = node.parm("res2").eval()
```

Comment:
Not sure why this was changed, but the old logic made more sense where it took the `rop_node` directly. Especially because the instance node already seems to be retrieved in `process`

Reply:
maybe thid is the issue coming from merging develop to the branch. i agree that `rop_node` is more concise than `node`